Snowdog_Components/components/Molecules/form/datepicker/_datepicker.scss

```diff
@@ -0,0 +1,89 @@
+@import 'datepicker-variables';
+
+.ui-datepicker {
+    border: $datepicker__border;
+    background-color: $datepicker__background-color;
+
+    .ui-datepicker-prev {
+        top: $datepicker__arrow-top;
+
+        .ui-icon {
+            border-color: $datepicker__prev-icon-border-color;
+        }
+    }
+
+    .ui-datepicker-next {
+        top: $datepicker__arrow-top;
+
+        .ui-icon {
+            border-color: $datepicker__next-icon-border-color;
+        }
+    }
+
+    .ui-datepicker-title {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        flex-direction: row-reverse;
+
+        .ui-datepicker-month,
+        .ui-datepicker-year {
+            background-color: $datepicker__select-background-color;
+            height: $datepicker__select-height;
+            margin-right: $spacer;
```

Comment:
we should use `$datepicker_select-margin-right`. easier to change it on variable level in child theme